lib/src/Datepicker/Datepicker.tsx

```diff
@@ -57,7 +57,12 @@ export const Datepicker: React.FunctionComponent<DatepickerProps> = ({
 
     const getStringFromDate = React.useCallback(
         (d: Date, monthPicker: boolean): string => {
-            return isValidDate(d) ? d?.toISOString()?.substr(0, monthPicker ? 7 : 10) || "" : "";
+            if (isValidDate(d)) {
+                const tzoffset: number = new Date().getTimezoneOffset() * 60000;
```

Comment:
are you converting to a different timezone here ?

Reply:
No I'm adjusting the date in a way which removes any timezone offset

Reply:
so when a user selects a date would there be any information on what timezone that date has ?

Reply:
If the user passes a date with timezone info in it, the timezone info will be removed and only the date will be respected.

Reply:
i think it's a bit dangerous to do that, especially if you are adding data in a timezone and the data is saved in a server that have a different timezone

Reply:
Okay I actually found a better way of achieving this so I updated this part. No timezone modification needed.